04week/welcome-to-our-site/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Welcome to our site!</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      @media (max-width:576px){
```

Comment:
don't put styling in your index.html file

all styles should go in your styles.css